examples/test-coverage.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
   test-coverage:
     runs-on: ubuntu-latest
     env:
-      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Please keep the `GITHUB_PAT`, some packages need that.

Reply:
Packages installing dependencies from GitHub is the most common.

Reply:
I see ! Thanks !